test/TEST-20-NFS/server-99-default.link

```diff
@@ -0,0 +1,6 @@
+[Match]
```

Comment:
We should just call this server.link since it's not the default file nor does having a priority ( 99 ) make any sense since it is not being picked up and thirdly this is the only link file being included.

Reply:
true